atomix/cluster/src/main/java/io/atomix/raft/metrics/RaftRoleMetrics.java

```diff
@@ -73,4 +80,8 @@ public void observeHeartbeatInterval(final long milliseconds) {
   public static double getHeartbeatMissCount(final String partition) {
     return HEARTBEAT_MISS.labels(partition, partition).get();
   }
+
+  public void setElectionLatency(final long latencyMs) {
+    ELECTION_LATENCY.labels(partitionGroupName, partition).set(latencyMs);
```

Comment:
Please be aware that we need to calculate the seconds, see above line 77

Reply:
Ok it is a gauge I have overseen that. Do you have already an idea for the visualization?

Reply:
Histogram was not useful. That's why I used Gauge. It can be either a graph (similar to snapshot replication duration) or a bar graph (similar to startup time).

Reply:
Yeah I would probably prefer the bar thing, which you did for the start up.